src/js/graph/parsers/graphParserXml.js

```diff
@@ -86,6 +86,7 @@ GraphParserXml.prototype.parse = function(readableStream) {
                     }
                     break;
                 case 'intFeat':
+                case 'enumProp':
```

Comment:
Doesn't look like there are any `enumProp`s in the streams. We'll need to add an example in one of the stream to ensure it's parsed and serialized correctly.